### DISCUSSION TITLE
network mainnet - mumbai is not supported?

### DISCUSSION THREAD

**Discussion by btfdip** [0xPolygon/matic.js#303] [Q&A] [answered]
await posClient.init({
        network: 'mainnet',
        version: 'mumbai',
        parent: {
            provider: new HDWalletProvider(a_random_sk, mainnet_RPC),
            defaultConfig: {
                from : a_random_address
            }
        },
        child: {
            provider: new HDWalletProvider(a_random_sk, childRPC),
            defaultConfig: {
                from : a_random_address
            }
        }
    });

**Comment by btfdip** [comment]
can you make it clear what's current setting is? really hard to find it by google


what's the common env setting we can use for testing and online running?

**Comment by ujjwalguptaofficial** [comment, marked as answer]
for network mainnet, version is `v1`. I will add more in documentation.
